llvm/include/llvm/Analysis/IR2Vec.h

```diff
@@ -162,16 +163,25 @@ using BBEmbeddingsMap = DenseMap<const BasicBlock *, Embedding>;
 /// embeddings.
 class Vocabulary {
   friend class llvm::IR2VecVocabAnalysis;
+  // Slot layout:
+  // [0 .. MaxOpcodes-1]                                   => Instruction
+  // opcodes [MaxOpcodes .. MaxOpcodes+MaxCanonicalTypeIDs-1]      =>
+  // Canonicalized types [MaxOpcodes+MaxCanonicalTypeIDs .. end of operands) =>
```

Comment:
can you explain what canonicalized means here?

Reply:
Is it okay now?

Reply:
well now you're setting the bar too high 😄 
(so yes - thanks, it looks awesome)

add maybe a note about what canonicalization means, and whether the limits are LLVM IR-based or embedding related - for example MaxOpcodes, is it 67 (or how many LLVM opcodes there are)?